collections/_article/dev-snapshot-godot-4-5-dev-4.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Dev snapshot: Godot 4.5 dev 4"
+excerpt: One post-GodotCon snapshot coming up!
+categories: [pre-release]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-5-dev-4.webp
+image_caption_title: Ambidextro
+image_caption_description: A game by Majorariatto
+date: 2025-05-12 12:00:00
+---
+
+To everyone who ended up going to GodonCon this past week, we hope you had a safe journey home! For those of you that missed the fun, we've logged some of the highlights on the blog already—such as the long-awaited [web support for .NET](/article/live-from-godotcon-boston-web-dotnet-prototype/)—with more to come later in the week. With that now behind us, we can go full-steam ahead on our next development snapshot: 4.5 dev 4. Plenty of new features this time around, so early feedback and bug reports from early testing are once again strongly recommended.
```

Comment:
Can you add a sentence saying that GodotCon all talks were recorded and will be uploaded to our youtube channel in a few weeks? We keep getting asked about it 😓